Cargo.toml

```diff
@@ -18,7 +18,7 @@ serde_json = "1.0"
 reqwest = "~0.6"
 log = "0.3"
 quick-error = "1.1.0"
-url = "1.4"
+url = "1.5"
```

Comment:
Oh, that was from the rebase. Previously, it was `~1.4`, which broke builds, and so I switched it to `1.5`, but then someone had already switched to `1.4` in the meantime.